experimental/framework/tree-react-api/.eslintrc.cjs

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [require.resolve("@fluidframework/eslint-config-fluid/strict"), "prettier"],
+	parserOptions: {
+		// TODO: why does tree package not have to list test config here?
```

Comment:
I believe it has an `overrides` block that specifies the other config for test files.